RELEASE-NOTES.md

```diff
@@ -22,6 +22,8 @@
   * Previously created procedures or functions won't work with `deploy` command due to change in stage path of artifact. Previous code will remain under old path on stage.
   * Package commands are now under `snow snowpark package`.
   * Coverage commands were removed. To measure coverage of your procedures or functions use coverage locally.
+  * Add `default` field to procedure and function arguments definition in `snowflake.yml` to support [named and optional
+  arguments](https://docs.snowflake.com/en/developer-guide/udf/udf-calling-sql#calling-a-udf-that-has-optional-arguments)
```

Comment:
Let's remember to add this to documentation

Reply:
I've reached out to Mike on Slack and in github comments (updating google doc accordingly)